docs/getting-started/customizing-your-setup.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Customizing your setup
+---
+
+# Customizing your setup
```

Comment:
To simplify, let's assume that this initial onboarding is just for the first repository. After the first repository is correctly set up [we'll have more guidance for the organization-level onboarding](https://codacy.atlassian.net/browse/DOCS-471) (which will include adding and configuring the remaining repositories).

As such, I suggest that the title of this page is more connected to the act of configuring this first repository. Here are two suggestions:

> Configuring your repository

> Customizing your repository

Reply:
Makes sense.

I was looking into renaming the page to "Configuring your repository" and then wondered if renaming the page slightly to specify that it's the first repository would make sense:

* Codacy quickstart > Adding your first repository
* Customizing your setup > Configuring your first repository

Thoughts?